crates/bevy_pbr/src/material.rs

```diff
@@ -137,6 +147,19 @@ pub trait Material: Asset + AsBindGroup + Clone + Sized {
         ShaderRef::Default
     }
 
+    /// Returns this material's deferred vertex shader. If [`ShaderRef::Default`] is returned, the default deferred vertex shader
+    /// will be used.
+    fn deferred_vertex_shader() -> ShaderRef {
+        ShaderRef::Default
```

Comment:
Does it make sense to fallback to `Self::vertex_shader()` instead of `ShaderRef::Default`?

If I understand it correctly the fragment shader will always be different for deferred rendering, but most of the time the vertex shader should be the same between forward and deferred?

Reply:
Yes. I think this would probably make sense. Also I think usually the fragment shader would also be the same, so it probably makes sense to default to that too. 

(Regarding sharing the fragment shader see the `StandardMaterial` and https://github.com/DGriffin91/bevy_glowy_orb_tutorial/blob/deferred/assets/shaders/glowy.wgsl#L56)

Reply:
Actually, this would want the prepass vertex shader, not the normal one. But then for the fragment it would likely want the normal one not the prepass. Maybe it should just be left up to the user I'm not totally sure.